MergerService/Properties/launchSettings.json

```diff
@@ -14,7 +14,11 @@
       "launchBrowser": true,
       "launchUrl": "swagger",
       "environmentVariables": {
-        "ASPNETCORE_ENVIRONMENT": "Development"
+        "ASPNETCORE_ENVIRONMENT": "Development",
+        "AWS_ACCESS_KEY_ID": "minio123",
+        "AWS_SECRET_ACCESS_KEY": "minio123",
+        "S3__url": "http://localhost:9000",
```

Comment:
"AWS_REGION": "custom", ?

Reply:
Doesn't matter... these are some launch settings for Rider